packages/tocco-ui/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "devDependencies": {
     "@fortawesome/fontawesome-svg-core": "^1.2.8",
+    "@fortawesome/free-brands-svg-icons": "^5.5.0",
```

Comment:
Though icons are chunked and delivered separately, it has an impact on loading time. As I added fortawesome I have decided not to include brands. Are brand icons requested now? How many brand icons are used? We still have the possibility to assemble an own set of brand icons.